batchENR.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2018 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package ach
+
+import (
+	"fmt"
+	"strings"
+	"unicode/utf8"
+)
+
+// BatchENR is a non-monetary entry that enrolls a person with an agency of the US government
+// for a depository financial institution.
+//
+// Allowed TransactionCode values: 22 Demand Credit, 27 Demand Debit, 32 Savings Credit, 37 Savings Debit
+type BatchENR struct {
```

Comment:
Each `Addenda05.PaymentRelatedInformation` contains the following data. Should we write a parser and validator for this `BatchENR` type? I started one with `EnrolleeClassificationCode ` below. 

```
// PaymentRelatedInformation (*) (\ ends)
// Example: 22*12200004*3*123987654321*777777777*DOE*JOHN*0\
//
// transaction code
// RDFI Identification Number (ABA)
// Check Digit
// DFI Account Number
// Individual Identification: SSN
// Individual Name
// Representive Payee Indicator / Enrollee Classification Code: Gov Benefits: 0 (no), otherwise 1 (yes - initiated by someone other than named beneficiary)
//   - A: enrollee is consumer,  B enrollee is a company
```

Reply:
Yes. CCD+ and CTX have something similar that is not currently implemented. I left it off because the data CAN be formatted but it doesn't have to be formatted in that way for commercial payments.

Reply:
I added a struct and parse method for this.